### DISCUSSION TITLE
How can I customize the components that are created with multiples components like Dialog, Drawer, etc... in Chakra V3?

### DISCUSSION THREAD

**Discussion by ericksonholguin** [chakra-ui/chakra-ui#9167] [Q&A] [answered]
### Description

The current sizes of the dialog component width do not suit my needs and in the documentation I cannot find a way to add custom measurements, could you please tell me how to do it? The closest I have seen to this in the documentation is in the Recipes section, but the information added there is very poor.

<img width="1107" alt="image" src="https://github.com/user-attachments/assets/de91294b-b485-44d2-9a5b-42af821c6f05">

### Link to Reproduction

https://stackblitz.com/edit/chakra-ui-v3?file=src%2Fcomponents%2Fui%2Fdialog.tsx

### Steps to reproduce

2. Click on '...'
3. Scroll down to '...'

### Chakra UI Version

3.1.1

### Operating System

- [X] macOS
- [X] Windows
- [X] Linux

**Comment by segunadebayo** [comment, marked as answer]
Please follow the docs here:
https://www.chakra-ui.com/docs/theming/customization/recipes

We'll consider a customization example to every component.

**Reply by ericksonholguin** [reply]
@segunadebayo the module `@chakra-ui/react/anatomy` doesn't exist, when I try to import something from that module, the Text editor shows me an error
<img width="2200" alt="image" src="https://github.com/user-attachments/assets/9653edc5-2edc-44c1-b44e-79c366d436fa">

**Reply by odinnix** [reply]
@Eriickson @segunadebayo I'm having the same problem. 
I can import via:

`from import { cardAnatomy } from '@chakra-ui/react/anatomy';`

However I then get a runtime error 

`2:06:27 p.m. [vite] error while updating dependencies:
Error: ENOENT: no such file or directory, open '/Users/odinnix/source/dlms.app.fe/node_modules/@chakra-ui/anatomy/dist/index.mjs'`

This is with pnpm esnext/bundler
